src/ui/popup/disallowed.tsx

```diff
@@ -0,0 +1,69 @@
+import ClonedSong from '@/core/object/cloned-song';
+import { ManagerTab } from '@/core/storage/wrapper';
+import { Resource, Show, createMemo, createSignal } from 'solid-js';
+import styles from './popup.module.scss';
+import optionComponentStyles from '../options/components/components.module.scss';
+import { t } from '@/util/i18n';
+import MusicOff from '@suid/icons-material/MusicOffOutlined';
+import Send from '@suid/icons-material/Send';
+import { sendBackgroundMessage } from '@/util/communication';
+
+export default function Disallowed(props: { tab: Resource<ManagerTab> }) {
+	const [isLoading, setLoading] = createSignal(false);
+	const song = createMemo(() => {
+		const rawTab = props.tab();
+		if (!rawTab) return null;
+		const rawSong = rawTab.song;
+		if (!rawSong) return null;
+		return new ClonedSong(rawSong, rawTab.tabId);
+	});
+
+	return (
+		<Show when={!isLoading()}>
+			<div class={styles.widePopup}>
+				<MusicOff class={styles.bigIcon} />
+				<h1>{t('disallowedHeader')}</h1>
+				<p>
+					{t('disallowedDesc1', [
+						song()?.getArtist() ?? '???',
+						song()?.getTrack() ?? '???',
+					])}
+				</p>
+				<Show
+					when={song()?.metadata.label === 'YouTube'}
```

Comment:
Indeed it’s not available currently. Since we are probably only ever using youtube, I think we can take having to change this if youtube were to rebrand in the future. If for some reason we end up adding more checks we can reevaluate then.

Reply:
True - I guess we don't need to over do it just now.